src/provider/rsa_asym_cipher.c

```diff
@@ -374,6 +386,19 @@ static int ibmca_asym_cipher_rsa_set_ctx_params(void *vctx,
     if (rc == 0)
         return 0;
 
+#ifdef OSSL_ASYM_CIPHER_PARAM_IMPLICIT_REJECTION
+    rc = ibmca_param_get_uint(ctx->provctx, params,
+                              OSSL_ASYM_CIPHER_PARAM_IMPLICIT_REJECTION,
+                              &implicit_rejection);
+    if (rc == 0)
+        return 0;
+    if (rc > 0 && implicit_rejection != 0) {
+        put_error_op_ctx(ctx, IBMCA_ERR_INVALID_PARAM,
+                          "RSA: Implicit rejection is not supported");
```

Comment:
Please check the intend of the second line. In github, it's off by one.

Reply:
I think he intends to get the indentation right ;-)

Reply:
fixed